src/common/xo/choose-sr-for-each-vdis-modal/index.js

```diff
@@ -32,80 +27,29 @@ Collapsible.propTypes = {
 }
 
 @propTypes({
-  vdis: propTypes.array.isRequired,
-  predicate: propTypes.func
+  mainSr: propTypes.string.isRequired,
+  mainSrPredicate: propTypes.func,
+  mapVdisSrs: propTypes.object.isRequired,
+  onChange: propTypes.func.isRequired,
+  srPredicate: propTypes.func,
+  vdis: propTypes.array.isRequired
 })
 export default class ChooseSrForEachVdisModal extends Component {
```

Comment:
Ok but the component is not a modal body, it's just a normal component used by a modal. It's probably not the right PR to change it, though.